src/i18n/i18n.js

```diff
@@ -0,0 +1,37 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { reactI18nextModule } from 'react-i18next';
+
+// individual language translation file
+import loadFr from './fr';
+import loadEn from './en';
+
+i18n
+  .use(LanguageDetector)
+  .use(reactI18nextModule)
+  .init({
+    // we init with resources
+    resources: {
+      en: loadEn(),
```

Comment:
Yes we can require a master `loadLanguages()` method instead. 
This method can be located in an `index.js` file inside a `locale` folder, so that we separate concerns.

This way when we add another language, we just provide `language_name.js`, add it into the index.js file as a key, and it will be available in i18next.